package.json

```diff
@@ -7,16 +7,25 @@
     "prettier-hook": "pretty-quick",
     "prettier-check-hook": "pretty-quick --check",
     "prepublishOnly": "make move-package-json-to-dist",
-    "update-version": "make update-entry"
+    "update-version": "make update-entry",
+    "test": "jest --env=jsdom"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.23.3",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
     "@rollup/plugin-node-resolve": "^8.0.1",
+    "@testing-library/react": "12.1.2",
```

Comment:
do we need all libs in dep? why not devdep?

Reply:
its on devdep